suite-common/wallet-core/src/blockchain/blockchainThunks.ts

```diff
@@ -315,6 +315,8 @@ export const unsubscribeBlockchainThunk = createThunk(
                         blocks: false,
                         accounts: accountIdentities[identity] ?? [],
                     }));
+
+                return [...transformedRemovedAccounts, { symbol, blocks: true, accounts: [] }];
```

Comment:
Added an account with no identity to removed accounts to unsubscribe both web sockets